_data/rdv.yml

```diff
@@ -0,0 +1 @@
+version: 7.4.0
```

Comment:
I assume this is the value that needs to get updated for new rdv releases?

Reply:
no actually, this file gets updated automatically by CI and will be untracked in a later pr when i refactor CI to be python instead of bash

Reply:
ooh sick